eosmetrics/emtr-aggregate-timer.c

```diff
@@ -106,21 +107,20 @@ on_server_aggregate_timer_started_cb (GObject      *source_object,
 
   if (error)
     {
-      g_warning ("Error creating aggregate timer: %s", error->message);
+      if (!g_error_matches (error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
+        g_warning ("Error creating aggregate timer: %s", error->message);
       return;
     }
 
-  proxy = G_DBUS_PROXY (source_object);
-  connection = g_dbus_proxy_get_connection (proxy);
-  emer_aggregate_timer_proxy_new (connection,
```

Comment:
I think part of the goal of not constructing a proxy is to remove this extra async operation when the only thing we do with the proxy is call a method on it, which we can do perfectly well with just the object path. I wonder whether `emer_aggregate_timer_proxy_new_sync` is actually guaranteed to not do any blocking stuff when we pass all these `DO_NOT` flags.